arm-datalake-analytics/job/2016-11-01/swagger/job.json

```diff
@@ -700,7 +700,10 @@
           "enum": [
             "Warning",
             "Error",
-            "Info"
+            "Info",
```

Comment:
This will be a breaking change for strongly typed languages or languages that support enums.